docker/swarm.docker-compose.yaml

```diff
@@ -0,0 +1,411 @@
+  order: stop-first
+
+x-restart-policy:
+  &restart-policy
+  condition: any
+  window: 10s
+
+x-rollback-config:
+  &rollback-config
+  parallelism: 1
+  failure_action: pause
+  monitor: 10s
+  max_failure_ratio: 0
+#  order: start-first
+  order: stop-first
+
+x-healthcheck:
+  &healthcheck
+  test: ["CMD", "curl", "-f", "http://localhost/health"]
+  interval: 10s
+  timeout: 10s
+  retries: 3
+  start_period: 5s
+
+services:
+
+  datagrok:
+    image: datagrok/datagrok:latest
+    stop_grace_period: 2m
+    deploy:
+      replicas: 1
+      mode: replicated
+      endpoint_mode: dnsrr
+      placement: *datagrok-placement
+      update_config:
+        <<: *update-config
+        monitor: 2m
+      restart_policy:
+        <<: *restart-policy
+        window: 30s
+      rollback_config:
+        <<: *rollback-config
+        monitor: 30s
+      resources:
+        reservations:
+          cpus: '1'
+          memory: 200M
+    healthcheck:
+      <<: *healthcheck
+      test: ["CMD", "curl", "-f", "http://localhost:8080/api/admin/health"]
+      interval: 30s
+      start_period: 30s
+    environment:
+      GROK_MODE: "auto"
+      GROK_PARAMETERS: |
+        {
+          "name": "prod",
+          "deployDemo": false,
+          "deployTestDemo": false,
+          "testTracking": false,
+          "createBetaUsers": true,
+          "debug": false,
+          "dbServer": "database",
+          "db": "datagrok",
+          "dbAdminLogin": "db_admin",
+          "dbAdminPassword": "db_password",
+          "dbLogin": "datagrok",
+          "dbPassword": "datagrok123",
+          "adminPassword": "admin",
+          "isolatesCount": 4
+        }
+    ports:
+      - target: 8080
+        published: 8080
+        protocol: tcp
+        mode: host
+    networks:
+      datagrok:
+        aliases:
+          - datagrok
+    volumes:
+      - datagrok_data:/home/grok/data
+      - datagrok_cfg:/home/grok/cfg
+    logging:
+      <<: *default-logging
+      options:
+        max-size: "1G"
+        max-file: "3"
+
+  grok_spawner:
+    image: datagrok/grok_spawner:latest
+    user: root
+    stop_grace_period: 2m
+    deploy:
+      replicas: 1
+      mode: replicated
+      endpoint_mode: dnsrr
+      placement:
+        <<: *datagrok-placement
+        constraints:
+          - "node.role==manager"
+          - "node.labels.role==datagrok"
+      update_config:
+        <<: *update-config
+        order: start-first
+      restart_policy:
+        <<: *restart-policy
+      rollback_config:
+        <<: *rollback-config
+        order: start-first
+      resources:
+        reservations:
+          cpus: '0.25'
+          memory: 200M
+    healthcheck:
+      <<: *healthcheck
+      test: ["CMD", "wget", "-SqO", "-", "http://localhost:8000/info"]
+    environment:
+      X_API_KEY: "x-api-key"
+      DOCKER_REGISTRY: "docker-registry"
+      DOCKER_REGISTRY_USER: "docker-registry-user"
+      DOCKER_REGISTRY_PASS: "docker-registry-password"
```

Comment:
It is better to remove, cause we do not mention registry in instructions. Or we need to add it.